### DISCUSSION TITLE
how can I install the Medusa Admin App seperately

### DISCUSSION THREAD

**Discussion by talha410900** [medusajs/medusa#5633] [Feature Requests]
I want to install the admin panel separately to make modification like branding, logo etc

**Comment by shavidze** [comment]
@shahednasser please, help us

**Comment by alightinastorm** [comment]
So far doesn't look like there's a way.

You can always clone the repo, extract the admin app and run that separately.
Though last time i've checked there were a ton of typescript errors, but that could be due to a misconfigured tsconfig (on my end)
